geomstats/geometry/skew_symmetric_matrices.py

```diff
@@ -0,0 +1,51 @@
+"""Module providing the SkewSymmetricMatrices class.
+
+This is the Lie algebra of the Special Orthogonal Group.
+As basis we choose the matrices with a single 1 on the upper triangular part
+of the matrices (and a -1 in int lower triangular part).
+"""
+import geomstats.backend as gs
+from geomstats.geometry.lie_algebra import MatrixLieAlgebra
+
+
+class SkewSymmetricMatrices(MatrixLieAlgebra):
+    """Implementation of skew symmetric matrices."""
+
+    def __init__(self, n):
+        """Instantiate the class.
+
+        Parameters
+        ----------
+        n: int
+            the amount of columns / rows
+        """
+        dimension = int(n * (n - 1) / 2)
+        super(SkewSymmetricMatrices, self).__init__(dimension, n)
+
+        self.basis = gs.zeros((dimension, n, n))
+        loop_index = 0
+
+        for i in gs.arange(n - 1):  # rows
```

Comment:
use i_row instead of inline comments, let's try to avoid comments in the code (not easily maintanable) and one-letter variables (complicated to do automatic replace)